operators/pkg/utils/common.go

```diff
@@ -59,7 +56,7 @@ func CheckSelectorLabel(ctx context.Context, k8sClient client.Client, namespaceN
 			return false, nil
 		}
 	} else {
-		return false, fmt.Errorf("error when retrieving the InstanceSnapshot namespace -> %w", err)
+		return false, fmt.Errorf("error when retrieving namespace <%s>: %w", namespaceName, err)
```

Comment:
```suggestion
		return false, fmt.Errorf("error when retrieving namespace %q: %w", namespaceName, err)
```

That takes care of automatic quoting